scripts/install/rtorrent.sh

```diff
@@ -64,13 +64,22 @@ Type=forking
 KillMode=none
 User=%i
 ExecStartPre=-/bin/rm -f /home/%i/.sessions/rtorrent.lock
+ExecStartPre=-/usr/local/bin/vmtouch -i '*.torrent' -m 90K -dl /srv/rutorrent/
+ExecStartPre=-/usr/local/bin/vmtouch -I '*.torrent.libtorrent_resume' -I '*.torrent.rtorrent' -m 5K -dl /home/%i/.sessions/
```

Comment:
I noticed my vmtouch processes are piling up each time the service is restarted. Adding something like this is probably a good idea:

`ExecStopPost=pkill -f -u %i '/usr/local/bin/vmtouch.*/home/%i/.sessions'`